app/assets/javascripts/to_do_lists.coffee

```diff
@@ -1,3 +1,62 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+# permit functions for eval
+permit_functions = ['expand_on_click', 'collapse_on_click']
+
+# hide element
+hide_element = (element) ->
+  element.addClass('hidden')
+
+# show element
+show_element = (element) ->
+  element.removeClass('hidden')
```

Comment:
If this is just to show/hide element you can use jQuery API: http://api.jquery.com/toggle/